src/components/installedAppsView/InstalledAppsView.js

```diff
@@ -34,6 +34,18 @@ class InstalledAppsView extends React.Component {
     return null;
   }
 
+  static getOpenshiftConsole(index) {
+    return (
+      <li
+        onClick={() => window.open(`${window.OPENSHIFT_CONFIG.masterUri}/console`, '_blank')}
+        key={`openshift_console_${index}`}
+        value={index}
+      >
+        Openshift Console
```

Comment:
small thing: `Openshift Console` -> `OpenShift Console`

Reply:
@jasonmadigan this is fixed now